CompanyAddress.md

```diff
@@ -7,124 +7,73 @@ _______
+|-------------------|--------|----------|
+|`company_id`      | string | True      |
+|`country` | string | False       |
+|`state` | string | False       |
+|`city` | string | False       |
+|`branch` | string | False       |
+|`postal_code` | string | False       |
 
-Each element of the returned **designated_market_areas** array will be formatted as follows:
+Example cURL request:
 
-|   |   |
-|---|---|
-| name | The name of the DMA, e.g. "Atlanta, GA." **This property is always present and non-empty.** |
-| id | The integer ID of the DMA. E.g. for the Atlanta, GA DMA, the ID is 524. **This property is always present and non-empty.** |
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/geography/companyaddress \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+          "postal_code": "53211",
+          "company_id": "NC496cd8bb-93bf-4fe1-b9fc-4c966f4f95e5",
+          "city": "Milwaukee",
+          "state": "WI",
+          "branch": "10th Street Comprehensive Treatment Center",
+          "country": "US"
+      }'
+```
 
-&nbsp;
 
-## Sample Response
 
+## Response Structure
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node. The response consists of company address and other location parameters like country, state, city, postal code, latitude, longitude and branch codes as show in the below sample response. The latitude and longitude are optional fields.
+### Sample Response
 ```json
 {
-    "data": {
-        "results": [
-            {
-                "admin_areas": [
-                    {
-                        "long_name": "Harju maakond",
-                        "short_name": "Harju maakond",
-                        "level": 1
-                    }
-                ],
-                "country": "Estonia",
-                "country_code": "EE",
-                "formatted_address": "Pikk 43, 10133 Tallinn, Estonia",
-                "latitude": 59.4396784,
-                "longitude": 24.7469663,
-                "locality": "Tallinn",
-                "location_type": "STREET_ADDRESS",
-                "postal_code": "10133",
-                "street_address": "43 Pikk",
-                "sublocality": "Kesklinn",
-                "place_id": "ChIJVZfruGOTkkYR4Ci9TcrEXFQ",
-                "viewport": {
-                    "northeast": {
-                        "lat": 59.4410273802915,
-                        "lng": 24.7483152802915
-                    },
-                    "southwest": {
-                        "lat": 59.43832941970849,
-                        "lng": 24.7456173197085
-                    },
-                    "suggested_radius_miles": 0.10459088520608802
-                },
-                "partial_match": false
-            }
-        ],
-        "expiration_date": "2018-04-22"
-    }
+  "data": {
+    "id": "NC496cd8bb-93bf-4fe1-b9fc-4c966f4f95e5",
```

Comment:
Is this the same Id that the client provides in the request? If the client already has this info is it valuable to return it?

Reply:
@glpayson : Need ur help here

Reply:
I think it is okay. It is coming back from the lucene index, so we would need to specifically exclude it. Also, we do this sort of thing when doing a place id lookup with the geocoding service, so it isn't like this is without precedent:

```
[~] curl -X POST \
  'https://wwwtest.api.careerbuilder.com/core/geography/geocoding?culture=en&place_id=ChIJd8BlQ2BZwokRAFUEcm_qrcA' \
  -H 'Accept: application/json;version=1.1' \
  -H 'Authorization: Bearer tb19UUIzlq-3OFH1by2OEPa3wve0HR6bbAIpxPa3buHEfBcI5xjPBrjckKHliw_8HqGvhu0pwLCL74dY9BaoeBvZmr2QfNDWOCRdhU20qY_HlRPC6_Zs2wXNmCjlWpLOej_mL1gXQn4fJbVxvrzejrGSl3KHfD2DaZV6UMNe8DXy-9Q=='
{
  "data": {
    "results": [
      {
        "admin_areas": [
          {
            "long_name": "New York",
            "short_name": "NY",
            "level": 1
          },
          {
            "long_name": "Kings County",
            "short_name": "Kings County",
            "level": 2
          }
        ],
        "country": "United States",
        "country_code": "US",
        "formatted_address": "277 Bedford Ave, Brooklyn, NY 11211, USA",
        "latitude": 40.7142205,
        "longitude": -73.9612903,
        "location_type": "STREET_ADDRESS",
        "postal_code": "11211",
        "street_address": "277 Bedford Avenue",
        "sublocality": "Brooklyn",
        **"place_id": "ChIJd8BlQ2BZwokRAFUEcm_qrcA"**,
        ...
```